src/Common/tests/System/PlatformDetection.cs

```diff
@@ -158,9 +158,13 @@ public static bool IsWinRT
         public static bool IsWindowsSubsystemForLinux => m_isWindowsSubsystemForLinux.Value;
         public static bool IsNotWindowsSubsystemForLinux => !IsWindowsSubsystemForLinux;
 
-        public static bool IsNotFedoraOrRedHatOrCentos => !IsDistroAndVersion("fedora") && !IsDistroAndVersion("rhel") && !IsDistroAndVersion("centos");
+        public static bool IsNotFedoraOrRedHatFamily => !IsDistroAndVersion("fedora") && !IsRedHatFamily;
```

Comment:
Use IsFedora instead of IsDistroAndVersion("fedora"). enuse IsFedora is moved before this line

Reply:
pending